ecosystem/typescript/sdk/examples/typescript-esm/rotate_key.ts

```diff
@@ -0,0 +1,58 @@
+import { AptosAccount, FaucetClient, Network, Provider, HexString } from 'aptos';
+
+const NETWORK = Network.DEVNET;
+const FAUCET_URL = `https://faucet.${NETWORK}.aptoslabs.com`;
```

Comment:
We should probably at some point make faucet client also use network cc @0xmaayan